misk/src/main/kotlin/misk/web/WebModule.kt

```diff
@@ -69,6 +69,10 @@ class WebModule : KAbstractModule() {
     binder().addMultibinderBindingWithAnnotation<NetworkInterceptor.Factory, MiskDefault>()
         .to<MetricsInterceptor.Factory>()
 
+    // Traces requests as they work their way through the system.
```

Comment:
maybe comment what dependency this has in the binding ordering. e.g this must be bound before the `ExceptionHandlingInterceptor` because ...